feature-configs/demo/sctp/sctp_module_mc.patch.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker-rt
```

Comment:
We should create a separate role for sctp as it is orthogonal to worker-rt. So we need manifests for MCP worker-sctp that reference worker-sctp node role and update the MC here to use it.

Reply:
@MarSik is it fine if I do the sctp stuff in a separate PR?

Reply:
removed